signer/config_test.go

```diff
@@ -523,7 +543,7 @@ func TestCosignerPeersConfigValidate(t *testing.T) {
 					P2PAddr: "tcp://127.0.0.1:2223",
 				},
 			},
-			shares:    3,
+			shares:    2,
```

Comment:
Does the user need to specify the # of shares? Could we use `len(peers)` instead?

Reply:
updated to remove this requirement